src/rpc/rawtransaction.cpp

```diff
@@ -672,6 +672,16 @@ UniValue verifytxoutproof(const UniValue& params, bool fHelp)
 
 UniValue createrawtransaction(const UniValue& params, bool fHelp)
 {
+    if (!fEnableCreateRawTransaction) {
+        throw runtime_error(
+            "createrawtransaction is DEPRECATED and will be removed in a future release.\n"
+            "\nZallet will instead have new RPC methods that work on PCZTs:\n"
+            "https://github.com/zcash/wallet/issues/99\n"
+            "Restart with `-allowdeprecated=createrawtransaction` if you require backward\n"
+            "compatibility.\n"
```

Comment:
Somewhat trivial suggestion: this isn't really "backward compatibility"; I would perhaps phrase it "to re-enable this method during its deprecation period".

EDIT: I just saw below that "if you require backward compatibility" is how this is phrased elsewhere, so keeping this as-is is fine, unless you wanted to make the change consistently across the deprecated methods.

Reply:
I agree and I'll make that change consistently.